flow/config.h.cmake

```diff
@@ -0,0 +1,50 @@
+#cmakedefine ALLOC_INSTRUMENTATION
+#cmakedefine NDEBUG
+#cmakedefine FDB_RELEASE
+#ifdef FDB_RELEASE
+# define FDB_CLEAN_BUILD
+#endif // FDB_RELEASE
+#cmakedefine OPEN_FOR_IDE
+#ifdef WIN32
+# define USE_USEFIBERS
+# define _WIN32_WINNT ${WINDOWS_TARGET}
+# define WINVER ${WINDOWS_TARGET}
+# define NTDDI_VERSION 0x05020000
+# define BOOST_ALL_NO_LIB
+#else
+# cmakedefine USE_UCONTEXT
```

Comment:
This should be unconditional, right? Not dependent on whether on the value of USE_UCONTEXT in cmake?

Reply:
yes, you're right. Will change this.